03-talos.tf

```diff
@@ -32,7 +34,9 @@ module "talos_worker_group" {
   subnet_id                   = element(data.aws_subnets.public.ids, tonumber(trimprefix(each.key, "${each.value.name}.")))
   associate_public_ip_address = true
   tags                        = merge(each.value.tags, var.tags, local.cluster_required_tags)
-
+  metadata_options            = var.metadata_options
```

Comment:
Remove this change, as it's already covered in your other PR: https://github.com/isovalent/terraform-aws-talos/pull/171